src/main/java/com/spotify/reaper/ReaperApplicationConfiguration.java

```diff
@@ -33,10 +36,12 @@
 
   @JsonProperty
   @NotNull
-  private String repairParallelism;
+  private RepairParallelism repairParallelism;
 
   @JsonProperty
   @NotNull
+  @Min(0)
```

Comment:
@Bj0rnen sorry I missed that comment in the YAML somehow. Looks like the `@Min(0)` could be replaced with `@DecimalMin(value="0", inclusive=false)` to only allow values in the range `(0, 1]`. I'll add that now.